simpletransformers/ner/ner_model.py

```diff
@@ -704,10 +718,16 @@ def train(
                         outputs = model(**inputs)
                         # model outputs are always tuple in pytorch-transformers (see doc)
                         loss = outputs[0]
+                        if self.loss_fct:
+                            logits = outputs[1]
+                            loss = self.loss_fct(logits.view(-1, self.num_labels), inputs['labels'].view(-1))
```

Comment:
```suggestion
                        if self.loss_fct:
                            logits = outputs[1]
                            labels = inputs['labels']
                            if inputs.get('attention_mask') is not None:
                                active_loss = attention_mask.view(-1) == 1
                                active_logits = logits.view(-1, self.num_labels)
                                active_labels = torch.where(
                                    active_loss, labels.view(-1), torch.tensor(loss_fct.ignore_index).type_as(labels)
                                )
                                loss = loss_fct(active_logits, active_labels)
                            else:
                                loss = loss_fct(logits.view(-1, self.num_labels), labels.view(-1))
```

@Zhylkaaa I understand and I think you are right. Look at the code in the suggestion, are you ok with that?

But I don't think this pull request will be merge because it was opened a long time ago and noone reviews or comment.

Reply:
yes, I think this will work. Also I hope that this changes will be merged :) 
There are still some commits to master so I guess library is alive 🤞

Reply:
Sorry this took so long! I'll merge this shortly.